pl_bolts/models/self_supervised/simclr/transforms.py

```diff
@@ -129,8 +119,24 @@ def __init__(
         )
 
 
-@under_review()
 class SimCLRFinetuneTransform:
```

Comment:
Can this somehow be also subclass of `SimCLRTrainTransform`? Seems like quite a bit of code is duplicated

Reply:
Yes, will do.